templates/commands/render.go

```diff
@@ -235,10 +261,21 @@ func (r *realFS) WriteFile(name string, data []byte, perm os.FileMode) error {
 }
 
 func (r *Render) Run(ctx context.Context, args []string) error {
-	if err := r.parseFlags(args); err != nil {
+	rf := &RenderFlags{}
+	if err := rf.Parse(args); err != nil {
 		return err
 	}
 
+	return r.RunWithFlags(ctx, rf)
+}
+
+// RunWithFlags renders a template according to the given flags, taking the
+// template form rf.Source and producing output in rf.Dest. This should be
```

Comment:
```suggestion
// template from rf.Source and producing output in rf.Dest. This should be
```

Reply:
I'll fix this in a followup PR so I can keep my approval bit 😄  thanks!